### ISSUE TITLE
Public Cloud available in OVHcloud Trusted Zone offer

### ISSUE THREAD

**Issue by JacquesMrz** [ovh/public-cloud-roadmap#180]
As our European most sensitive data and critical workloads are targeted by cyber threats from organized criminals and government interference, it require specific level of protection.

OVHcloud is offering a new range of products dedicated to most strategic European organisations cloud services, hosted in "Trusted Zones" isolated from other OVHcloud standard services, offering the highest levels of physical security for client data (SecNumCloud and C5 levels), and dedicated professional services to answer clients custom security requirements.

Every service offered in the "Trusted Zones" will be operated from EU only, providing a complete immunity to extraterritorial laws, and will offer the highest levels of security compliance. Highest level of support level (Enterprise Support) is required for every Trusted Zones services.

Every service available in the Trusted Zone offers specific SLA, complying with ISO 27001. Some like Hosted Private Cloud services are already compliant with highest security standards like SecNumcloud and C5. Some others like "Bare Metal" and "Public Cloud" will be offered without SecNumCloud or C5 compliance and will get compliant in the future

All Trusted Zones services are aiming to comply in the future "High level" ENISA EUCS cloud security certification, leveraging especially on SecNumCloud and C5 standards.

https://www.ovhcloud.com/fr/enterprise/trusted-zone/

---

In other words, for Public Cloud, Trusted Zone offer will be hosted in a dedicated OpenStack Region, relying on a dedicated Keystone. Run/maintenance will be handled in Europe only.
ETA: end of February 2022

Public Cloud Services available at launch:
Compute
Block Storage
Object Storage
Volume Snapshot
Instance Backup
Private Network
Security Groups

**Comment by tanandy** [comment]
Hi, do you plan to support MKS too ?

**Comment by mhurtrel** [comment]
Hi @tanandy !
 Yes we plan to support MKS in this zone at a later stage. I can't yet share an ETA.

**Comment by JacquesMrz** [comment]
To all,
Any Public Cloud Services that would be missing in Trusted Zone, which you would need ?
Please share it here.

**Comment by tanandy** [comment]
Hi, Managed services, DbaaS , Container registry , KMS ... same as public but in private
